Mathlib/AlgebraicTopology/DoldKan/Compatibility.lean

```diff
@@ -51,145 +51,149 @@ namespace DoldKan
+@[simps!]
+def equivalence₂CounitIso : (eB.functor ⋙ e'.inverse ⋙ eA.inverse) ⋙ F ⋙ eB.inverse ≅ 𝟭 B :=
   calc
-    (eB.Functor ⋙ e'.inverse ⋙ eA.inverse) ⋙ F ⋙ eB.inverse ≅
-        eB.Functor ⋙ (e'.inverse ⋙ eA.inverse ⋙ F) ⋙ eB.inverse :=
+    (eB.functor ⋙ e'.inverse ⋙ eA.inverse) ⋙ F ⋙ eB.inverse ≅
+        eB.functor ⋙ (e'.inverse ⋙ eA.inverse ⋙ F) ⋙ eB.inverse :=
       Iso.refl _
-    _ ≅ eB.Functor ⋙ 𝟭 _ ⋙ eB.inverse :=
+    _ ≅ eB.functor ⋙ 𝟭 _ ⋙ eB.inverse :=
       (isoWhiskerLeft _ (isoWhiskerRight (equivalence₁CounitIso hF) _))
-    _ ≅ eB.Functor ⋙ eB.inverse := (Iso.refl _)
+    _ ≅ eB.functor ⋙ eB.inverse := (Iso.refl _)
     _ ≅ 𝟭 B := eB.unitIso.symm
-    
 #align algebraic_topology.dold_kan.compatibility.equivalence₂_counit_iso AlgebraicTopology.DoldKan.Compatibility.equivalence₂CounitIso
 
 theorem equivalence₂CounitIso_eq : (equivalence₂ eB hF).counitIso = equivalence₂CounitIso eB hF :=
   by
   ext Y'
-  dsimp [equivalence₂, iso.refl]
-  simp only [equivalence₁_counit_iso_eq, equivalence₂_counit_iso_hom_app,
-    equivalence₁_counit_iso_hom_app, functor.map_comp, assoc]
+  dsimp [equivalence₂, Iso.refl]
+  simp only [equivalence₁CounitIso_eq, equivalence₂CounitIso_hom_app,
+    equivalence₁CounitIso_hom_app, Functor.map_comp, assoc]
 #align algebraic_topology.dold_kan.compatibility.equivalence₂_counit_iso_eq AlgebraicTopology.DoldKan.Compatibility.equivalence₂CounitIso_eq
 
 /-- The unit isomorphism of the equivalence `equivalence₂` between `A` and `B`. -/
-@[simps]
-def equivalence₂UnitIso : 𝟭 A ≅ (F ⋙ eB.inverse) ⋙ eB.Functor ⋙ e'.inverse ⋙ eA.inverse :=
+@[simps!]
+def equivalence₂UnitIso : 𝟭 A ≅ (F ⋙ eB.inverse) ⋙ eB.functor ⋙ e'.inverse ⋙ eA.inverse :=
   calc
     𝟭 A ≅ F ⋙ e'.inverse ⋙ eA.inverse := equivalence₁UnitIso hF
     _ ≅ F ⋙ 𝟭 B' ⋙ e'.inverse ⋙ eA.inverse := (Iso.refl _)
-    _ ≅ F ⋙ (eB.inverse ⋙ eB.Functor) ⋙ e'.inverse ⋙ eA.inverse :=
+    _ ≅ F ⋙ (eB.inverse ⋙ eB.functor) ⋙ e'.inverse ⋙ eA.inverse :=
       (isoWhiskerLeft _ (isoWhiskerRight eB.counitIso.symm _))
-    _ ≅ (F ⋙ eB.inverse) ⋙ eB.Functor ⋙ e'.inverse ⋙ eA.inverse := Iso.refl _
-    
+    _ ≅ (F ⋙ eB.inverse) ⋙ eB.functor ⋙ e'.inverse ⋙ eA.inverse := Iso.refl _
 #align algebraic_topology.dold_kan.compatibility.equivalence₂_unit_iso AlgebraicTopology.DoldKan.Compatibility.equivalence₂UnitIso
 
 theorem equivalence₂UnitIso_eq : (equivalence₂ eB hF).unitIso = equivalence₂UnitIso eB hF := by
   ext X
   dsimp [equivalence₂]
-  simpa only [equivalence₂_unit_iso_hom_app, equivalence₁_unit_iso_eq,
-    equivalence₁_unit_iso_hom_app, assoc, nat_iso.cancel_nat_iso_hom_left]
+  simp only [equivalence₂UnitIso_hom_app, equivalence₁UnitIso_eq, equivalence₁UnitIso_hom_app,
+      assoc, NatIso.cancel_natIso_hom_left]
+  rfl
 #align algebraic_topology.dold_kan.compatibility.equivalence₂_unit_iso_eq AlgebraicTopology.DoldKan.Compatibility.equivalence₂UnitIso_eq
 
 variable {eB}
 
-include hG
+-- porting note: commented out include
+-- include hG
```

Comment:
Just remove.

Reply:
(and the other two)

Reply:
make sense..
I saw you already did it just now